cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -101,8 +95,12 @@ public Boolean call() throws Exception {
         if (!getMonitorClient().allSystemServicesOk()) {
           return false;
         }
-        // check that the dataset service is up
-        getNamespaceClient().list();
+        // check that the dataset service is up, and also that the default namespace exists
+        try {
+          getNamespaceClient().get(Id.Namespace.DEFAULT);
```

Comment:
is this needed? if dataset service is not up, other services would not be up right ?